xenserver/provision.erb

```diff
@@ -0,0 +1,32 @@
+<%#
+kind: provision
+name: XenServer default answerfile
+oses:
+- XenServer
+%>
+<% subnet = @host.subnet -%>
+<% dhcp = subnet.dhcp_boot_mode? -%>
+<% interface_name = @host.primary_interface.identifier.empty? ? 'eth0' : @host.primary_interface.identifier -%>
```

Comment:
@ares is `@host.primary_interface.identifier` correct? this didn't work for me, but could be related to my installation